packages/netsuite-adapter/src/elements_source_index/elements_source_index.ts

```diff
@@ -89,6 +89,7 @@ const createIndexes = async (elementsSource: ReadOnlyElementsSource):
   const customFieldsIndex: Record<string, InstanceElement[]> = {}
   const pathToInternalIdsIndex: Record<string, number> = {}
   const elemIdToChangeByIndex: Record<string, string> = {}
+  const mapKeyFieldsIndex: Record<string, string> = {}
```

Comment:
Should it be like below?
```suggestion
  const mapKeyFieldsIndex: Record<string, string | string[]> = {}
```